Target-MVD-Mathias/ViewControllers/SignInViewController.swift

```diff
@@ -79,7 +123,7 @@ class SignInViewController: UIViewController {
     
     UserAPI.login(email, password: password, success: { (_) in
       self.hideSpinner()
-      UIApplication.shared.keyWindow?.rootViewController = UIStoryboard.instantiateViewController(HomeViewController.self, storyboardIdentifier: "Onboarding")
+      UIApplication.shared.keyWindow?.rootViewController = UIStoryboard.instantiateViewController(HomeViewController.self, storyboardIdentifier: "Main")
```

Comment:
The default value for the parameter `storyboardIdentifier ` is "Main" so you can delete this from the method